build/build/src/enso.rs

```diff
@@ -121,6 +122,7 @@ impl BuiltEnso {
         ENSO_META_TEST_ARGS.set(&format!("{} --run", ir_caches.flag()))?;
 
         ENSO_ENABLE_ASSERTIONS.set("true")?;
+        ENSO_TEST_ANSI_COLORS.set("true")?;
```

Comment:
This just makes the output of the tests more colorful. Like so:
![image](https://github.com/enso-org/enso/assets/14013887/9c73cc63-dac8-42d4-81e9-57fc66d72fc8)
(Inspired by the output from ScalaTest framework)